### PR TITLE
remove sub-modules and reorganise

### PR DESCRIPTION
**Pull request by gsteenkamp89**

### CODE REVIEW

File: src/core/core.tsx [hunk 1]
```diff
@@ -179,24 +177,6 @@ export const blockExplorerTransactionLink = (
   return chainIdToNetwork[chainID].getExplorerTransactionLink(transactionHash);
 };
 
-// pass in [JSON].abi
```
Comment (gaelicgod) [inline src/core/core.tsx:182]: Why is this being deleted?

File: src/core/utils.ts [hunk 2]
```diff
@@ -83,25 +80,19 @@ export const uriToHttp = (uri: string): string[] => {
...
+export const getEnsNameFromAddress = (address: string, library: any) => {
+  if (utils.isAddress(address)) {
     return library.lookupAddress(address);
   } else return null;
 };
 
 // a type guard for MM specific methods/properties
 export const isMetaMaskProvider = (
-  library: MetaMaskProvider | JsonRpcProvider
+  library: any
```
Comment (gaelicgod) [inline src/core/utils.ts]: `any` typeGuard 😅

File: src/core/core.tsx (hunk 1 above)
Comment (gsteenkamp89) [reply src/core/core.tsx:182]: This is not being used since we're not loading up any contracts

File: src/core/utils.ts (hunk 2 above)
Comment (gsteenkamp89) [reply src/core/utils.ts]: For some reason the JsonRpcProvider interfaces from ethers and usedapp are incompatible not sure why

**Pull request merged**